src/app/app.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+
+@Pipe({
```

Comment:
Is this a good spot for a "core" module? I need to actually research what the best practices are for that, but it feels weird putting it at root of `src/app`

(Also this doesn't actually matter for the slides, but still want to check)